bc_obps/registration/management/commands/load_fixtures.py

```diff
@@ -1,11 +1,46 @@
-import pathlib
 from django.core.management.base import BaseCommand
 from django.core.management import call_command
 
 
 class Command(BaseCommand):
-    help = 'Load all fixtures in app directories'
+    help = 'Load fixtures for specified workflow or all fixtures if no argument provided'
 
-    def handle(self, *args, **kwargs):
-        cmd_args = list(pathlib.Path().glob('./registration/fixtures/*/*.json'))
-        call_command('loaddata', *cmd_args)
+    def add_arguments(self, parser):
+        parser.add_argument('workflow', nargs='?', type=str, help='Name of the workflow')
+
+    def handle(self, *args, **options):
+        fixture_base_dir = 'registration/fixtures/mock'
+
+        workflow_fixtures = {
+            "cas_admin": [
+                f'{fixture_base_dir}/address.json',
+                f'{fixture_base_dir}/user.json',
+                f'{fixture_base_dir}/contact.json',
+                f'{fixture_base_dir}/bc_obps_regulated_operation.json',
+                f'{fixture_base_dir}/operator.json',
+                f'{fixture_base_dir}/operation.json',
+                f'{fixture_base_dir}/user_operator.json',
+            ],
+            # Add more workflows here
```

Comment:
Thanks for setting this up! This PR went in: https://github.com/bcgov/cas-registration/pull/1095 so we'll need to add a workflow for it. It can be in a separate card/PR; just wanted to flag here so we don't forget.

here it will be:
```
"industry_user": [
                f'{fixture_base_dir}/address.json',
                f'{fixture_base_dir}/user.json',
                f'{fixture_base_dir}/contact.json',
                f'{fixture_base_dir}/bc_obps_regulated_operation.json',
                f'{fixture_base_dir}/operator.json',
                f'{fixture_base_dir}/operation.json',
            ],
```

and then in `industry_user.spec` this line gets deleted: `await deleteUserOperatorRecord(process.env.E2E_INDUSTRY_USER_GUID as string);`

Reply:
https://github.com/orgs/bcgov/projects/122/views/2?pane=issue&itemId=57255843